src/XmlScope.ts

```diff
@@ -52,48 +52,52 @@ export class XmlScope extends Scope {
     }
 
     private diagnosticValidateInterface(callableContainerMap: CallableContainerMap) {
-        if (!this.xmlFile.parser.ast?.component?.api) {
+        const component = this.xmlFile.parser.ast?.component;
+        //roku supports multiple <interface> elements, so check that there's at least one
+        if (!component || component.getChildNodesByTagName('interface').length === 0) {
```

Comment:
Why not:
```
const interfaceMembers = component?.interfaceMembers || [];
if (interfaceMembers.length === 0) { return; }
```

Reply:
Yeah, I have no idea why I did it this way! (perhaps I was just resolving tsc errors and didn't think through optimizing the code). This has been updated.